src/preact/amp-base-element.js

```diff
@@ -121,12 +121,12 @@ export class AmpPreactBaseElement extends PreactBaseElement {
  * Changes the inheritance hierarchy of X such that it now extends from Y.
```

Comment:
Worth updating the comment too, since X and Y are no longer the variable names.

Reply:
good call - done [here](https://github.com/ampproject/amphtml/compare/main...kvchari:fix-comment?expand=1)